src/frontend/ikoner/StatusIkon.tsx

```diff
@@ -27,7 +27,7 @@ export enum Status {
     OK,
 }
 
-const StatusIkon: React.FC<IProps> = ({ status, heigth, width }) => {
+const StatusIkon: React.FC<IProps> = ({ status, heigth = 24, width = 24 }) => {
```

Comment:
Legger inn defaultverdier for å unngå at vi brekker noe visuelt. Gamle Oppfylt/IkkeVurdert/IkkeOppfylt hadde denne størrelsen. Andre steder vi bruker StatusIkon har allerede overstyrt størrelsen til å være 20/20